Adding BabylonReactNative to a project already containing packages that require ARCore may fail to build

Adding BabylonReactNative to a project containing Viro React Community(https://github.com/ViroCommunity/viro) fails to build:

>[:arcore_client] /project/node_modules/react-viro/android/arcore_client/build/.transforms/8a1c382d15b6830d95b4fffc34498914/jetified-core-1.10.0/AndroidManifest.xml Warning:
Package name ‘com.google.ar.core’ used in: :arcore_client, com.google.ar:core:1.14.0.

>See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.

>Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use ‘–warning-mode all’ to show the individual deprecation warnings.
See Command-Line Interface
265 actionable tasks: 5 executed, 260 up-to-date
/project/android/app/src/debug/AndroidManifest.xml:35:13-38 Error:
Attribute meta-data#com.google.ar.core.min_apk_version@value value=(190519000) from [:arcore_client] AndroidManifest.xml:35:13-38
is also present at [com.google.ar:core:1.14.0] AndroidManifest.xml:35:13-38 value=(191106000).
Suggestion: add ‘tools:replace=“android:value”’ to element at AndroidManifest.xml:33:9-35:41 to override.

>FAILURE: Build failed with an exception.

>What went wrong:
Execution failed for task ‘:app:processDebugMainManifest’.
Manifest merger failed with multiple errors, see logs`

Both require ARCore 1.14.0, but Viro React targets Android SDK 24 whilst BabylonReactNative targets SDK 28.

--- a/Package/Android/build.gradle
+++ b/Package/Android/build.gradle
@@ -77,7 +77,6 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'  // From node_modules
-    implementation 'com.google.ar:core:1.14.0'
 }
 
 def configureReactNativePom(def pom) {

